python/lsst/ts/dimm/dimm_csc.py

```diff
@@ -259,7 +336,7 @@ async def health_monitor(self):
                         ),
                     )
 
-                    self.fault(code=SEEING_LOOP_DONE, report=error_report)
+                    await self.fault(code=SEEING_LOOP_DONE, report=error_report)
```

Comment:
I think `fault` is a regular method no a coroutine.

Reply:
When I remove `await` my IDE complains that it is a coroutine that is not awaited so I'll leave it.

Reply:
This is why:

https://github.com/lsst-ts/ts_dimm/blob/cac576a7b47cc2297591487c3097156ca6e9a2f6/python/lsst/ts/dimm/dimm_csc.py#L326-L334

the CSC "overrides" the method with a coroutine... Maybe should fix it?

Reply:
Good catch! Fixed.

Reply:
Thanks for fixing.. can you remove the await then?

Reply:
Done